2.55.0 版本新增的组件 `ColorPicker` 打包的样式没有提供对应 css 文件

- [x] I'm sure this does not appear in [the issue list of the repository](https://github.com/arco-design/arco-design-vue/issues)

## 基本信息 
- **依赖包名 及 版本:** @arco-design/web-vue@2.55.0
- **框架版本:** vue@3.4.21
- **浏览器版本:** chrome122.0.0.0

## 预期结果
正常使用 导入 `ColorPicker` 的 css 样式文件

## 复现步骤
导入 `import '@arco-design/web-vue/es/color-picker/style/css' `
报错
```
 [ERROR] Could not resolve "./index.css"

    node_modules/.pnpm/@arco-design web-vue@2.55.0_vue@3.4.21/node_modules/@arco-design/web-vue/es/color-picker/style/css.js:2:7:
      2 │ import "./index.css";
```
查看对应文件夹，的确未发现 `./index.css` 文件
<img width="219" alt="image" src="https://github.com/arco-design/arco-design-vue/assets/56616107/256e904f-a5ea-486a-9cd2-265b2d941a5b">


<!--- 声明: 如果提交带有攻击性的 issue，提交者将会被禁止在 arco-design 组织下发言。 -->
<!-- generated by arco-issue. DO NOT REMOVE -->

--- a/packages/web-vue/components/color-picker/style/index.ts
+++ b/packages/web-vue/components/color-picker/style/index.ts
@@ -1,2 +1,5 @@
 import '../../style/index.less';
+import '../../input/style';
+import '../../input-number/style';
+import '../../select/style';
 import './index.less';

--- a/packages/web-vue/components/color-picker/style/token.less
+++ b/packages/web-vue/components/color-picker/style/token.less
@@ -1,4 +1,5 @@
 @import '../../style/theme/index.less';
+@import '../../input/style/token.less';
 
 @color-preview-size-mini: @size-4;
 @color-preview-size-small: 22px;

